Client/game_sa/CModelInfoSA.cpp

```diff
@@ -1169,6 +1183,11 @@ void CModelInfoSA::ResetAlphaTransparency()
     }
 }
 
+void CModelInfoSA::StaticReset2DFXEffects()
+{
+
+}
```

Comment:
what about this?

Reply:
This PR is marked as draft, so it's not ready for review yet